docs/accessing-the-cluster.md

```diff
@@ -1,9 +1,27 @@
 # User Guide to Accessing the Cluster
- * [Accessing the cluster API](#api)
- * [Accessing services running on the cluster](#otherservices)
- * [So many proxies](#somanyproxies)
 
-## Accessing the cluster API<a name="api"></a>
+**Table of Contents**
+<!-- BEGIN GENERATED TOC -->
```

Comment:
This needs to become the new universal munge tag:

<!-- BEGIN MUNGE: GENERATED_TOC -->